hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/util/FakeTimer.java

```diff
@@ -29,17 +29,19 @@
 @InterfaceAudience.Private
 @InterfaceStability.Unstable
 public class FakeTimer extends Timer {
+  private long now;
   private long nowNanos;
 
   /** Constructs a FakeTimer with a non-zero value */
   public FakeTimer() {
     // Initialize with a non-trivial value.
+    now = 1577836800000L; // 2020-01-01 00:00:00,000+0000
```

Comment:
Can we leave the old behavior as it was and add a method to change the nowNanos so we only set it there?

Reply:
Thank you for the comment.

Actually, I think it's more real behavior than original one, because `Timer.now()` returns the time based on `System.currentTimeMillis()`, and `Timer.monotonicNow()` and `Timer.monotonicNowNanos()` return the time based on `System.nanoTime()`, which are different times. 

This change is useful to verify the fix in this PR. Also, it doesn't break the existing tests.

What do you think?